contributions/catalog/README.md

```diff
@@ -0,0 +1,105 @@
+# Contribution-Catalog
+
+## Run in Development Mode
+MongoDB's service needs to be started and a Mongo Shell needs to be connected in a separate terminal instance
+
+- For MacOS
+```
+brew services start mongodb-community@4.2
+mongo
+```
+
+- Ubuntu (The daemon-reload command isn't required)
+```
+sudo systemctl daemon-reload
+sudo systemctl start mongod
+mongo
+```
+
+This repository should be put inside `/catalog` and run outside the folder. 
+There are two ways to do so:
+
+The first one:
+
+- (Linux or MAC):
+```
+export FLASK_APP=catalog
+export FLASK_ENV=development
+flask run
+```
+
+- (Windows):
+```
+set FLASK_APP=catalog
+set FLASK_ENV=development
+flask run
+```
+
+The second one:
+
+- create a file called .flaskenv
+- fill in file with:
+    - FLASK_APP=\_\_init\_\_.py
+    - FLASK_ENV=development
+    - FLASK_DEBUG=1
+
+## Setup Environment and start the catalog application
+- (Linux or MAC):
+```
+cd catalog
+virtualenv -p python3 venv
+source venv/bin/activate
+pip install -r requirements.txt
+cd .. 
+set FLASK_APP=catalog
+set FLASK_ENV=development
+flask run --port=5050
+```
+- (Windows):
+```
+cd catalog
+py -m venv env
+.\env\Scripts\activate
+pip install -r requirements.txt
+cd .. 
+set FLASK_APP=catalog
+set FLASK_ENV=development
+flask run --port=5050
+```
+
+The following environment variables need to be set when running on development machine. 
+This is not required when running within AWS.
```

Comment:
I think that it will be better if you put this ahead of the actual running command explanation.